sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,222 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** be able to see the full art piece with detail information
+**I want to** view detailed information about an art piece
 
-**so that** I can learn everything about the piece.
+**so that** I can learn everything about it.
 
 #### Acceptance Criteria
 
-- [ ] Clicking an art piece from the list shows the detail page
-- [ ] The art piece image is displayed
-- [ ] The art piece title is displayed
-- [ ] The art piece artist is displayed
-- [ ] The art piece year is displayed
-- [ ] The art piece genre is displayed
-- [ ] A back-button is displayed
-- [ ] Clicking the back-button shows the list view
+- [ ] Clicking on an art piece from the list navigates to the detail page.
+- [ ] The art piece’s image is displayed.
+- [ ] The art piece’s title is displayed.
+- [ ] The art piece’s artist is displayed.
+- [ ] The art piece’s year is displayed.
+- [ ] The art piece’s genre is displayed.
+- [ ] A back button is displayed.
+- [ ] Clicking the back button navigates back to the list view.
 
 #### Tasks
 
-- [ ] Create the component `ArtPieceDetails`
-- [ ] `ArtPieceDetails` props: `image`, `title`, `artist`, `year`, `genre`
-- [ ] Create the page `pages/art-pieces/[slug]` that renders `ArtPieceDetails`
-- [ ] Read the query parameter `slug` from `next/router`
-- [ ] Use the `slug` to find the art piece to display
+- [ ] Create a new component for displaying Art Piece Details.
+- [ ] Create a single page that renders the Art Piece Details, displaying information for the selected art piece.
+- [ ] Write tests for the Art Piece Details component to confirm that all information is rendered correctly.
+
+### 5. Favorite Page
+
+The Favorite page will display a list of all favorited art pieces. Each art piece will show its image, artist’s name, title, and a favorite button.
 
----
+Users should be able to save their favorite art pieces both on the Gallery page and the Detail page.
 
-### User Story 5: Favorites
+To achieve this, we’ll divide the tasks into two parts:
+
+1. Implement the favorite button where needed.
+2. Create the Favorite page.
+
+### 5.1. Favorite Button
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** mark art piece pieces as favorites
+**I want to** mark art pieces as favorites
 
-**so that** I can find them easier.
+**so that** I can find them more easily.
 
 #### Acceptance Criteria
 
-- [ ] The favorite-button is displayed in each entry in the list view
-- [ ] The favorite-button is displayed in the details view
-- [ ] The favorite-button is NOT displayed in the spotlight view
-- [ ] Clicking the favorite-button on a non-favorite piece saves it as a favorite
-- [ ] Clicking the favorite-button on a favorite piece removes it from favorites
+- [ ] The favorite button is displayed on each entry in the list view.
+- [ ] The favorite button is displayed in the details view.
+- [ ] The favorite button is **NOT** displayed in the spotlight view.
```

Comment:
why not? I think it makes sense to keep the favorite button there as well.

Reply:
It was like that for NF. I didn't change it.
If you want we can change it, that the button to `The favorite button is displayed in the spotlight view.`

Reply:
yeah lets change that, it makes sense from a UX perspective and students can reuse the ArtpiecePreview component.